plugins/inventory/foreman.py

```diff
@@ -238,10 +296,201 @@ def _get_hostvars(self, host, vars_prefix='', omitted_vars=()):
                 hostvars[vars_prefix + k] = v
         return hostvars
 
+    def _fetch_params(self):
+        options = ("no", "yes")
+        params = dict()
+        report_options = {'want_location': True, 'want_organization': True, 'want_IPv4': True, 'want_IPv6': True, 'want_host_group': True, 'want_hostcollections': False, 'want_subnet': True, 'want_subnet_v6': True, 'want_smart_proxies': True, 'want_content_facet_attributes': False, 'want_host_params': True}
```

Comment:
```suggestion
        report_options = {'want_location': True, 'want_organization': True, 'want_ipv4': True, 'want_ipv6: True, 'want_host_group': True, 'want_hostcollections': False, 'want_subnet': True, 'want_subnet_v6': True, 'want_smart_proxies': True, 'want_content_facet_attributes': False, 'want_host_params': True}
```

it's lower case in the docs, so it should be lower case here

Reply:
and the linter complains: `E501 line too long (308 > 160 characters)` ;)